README.md

```diff
@@ -1,38 +1,81 @@
-Roadrunner for CakePHP
-===================
+# Roadrunner for CakePHP
 
 [![Downloads](https://poser.pugx.org/cakedc/cakephp-roadrunner/d/total.png)](https://packagist.org/packages/cakedc/cakephp-roadrunner)
 [![Latest Version](https://poser.pugx.org/cakedc/cakephp-roadrunner/v/stable.png)](https://packagist.org/packages/cakedc/cakephp-roadrunner)
 [![License](https://poser.pugx.org/cakedc/cakephp-roadrunner/license.svg)](https://packagist.org/packages/cakedc/cakephp-roadrunner)
 
+[RoadRunner](https://roadrunner.dev/) is a high-performance PHP application server, load-balancer, and process 
+manager written in Golang. Using Roadrunner you can replace php-fpm a long with nginx or apache.
 
-Requirements
-------------
+## Requirements
 
-* CakePHP 3.6.0+
-* PHP 7.1+
+* CakePHP ^4.3
+* PHP ^8.0
+* Roadrunner 2.7
 
-Setup
------
+## Table of Contents
+ 
+- [Install](#install)
+- [Sessions](#install)
+- [Static Assets](#static-assets)
+- [Support](#support)
 
-* `composer require cakedc/cakephp-roadrunner`
-* Download roadrunner binary and place the file in your filesystem, for example under `/usr/local/bin/rr`
-* Create a RoadRunner worker file, or use the example worker provided
+## Install
 
-```bash
+Install via composer:
+
+```console
+composer require cakedc/cakephp-roadrunner
+```
+
+Unlike most CakePHP plugins you won't be needing to load the plugin in your `src/Application.php`.
```

Comment:
Probably doesn't matter much but if this doesn't need to be loaded as a plugin then having type `cakephp-plugin` in composer.json is not necessary.

Reply:
I'd say just leave it.